packages/notifi-react-hooks/lib/hooks/useNotifiClient.ts

```diff
@@ -1509,6 +1509,22 @@ const useNotifiClient = (
     [setLoading, setError, service],
   );
 
+  const getUnreadNotificationHistoryCount = useCallback(async () => {
+    try {
+      const result = await service.getUnreadNotificationHistoryCount();
+      return result;
+    } catch (e: unknown) {
+      if (e instanceof Error) {
+        setError(e);
+      } else {
+        setError(new NotifiClientError(e));
+      }
+      throw e;
+    } finally {
+      setLoading(false);
+    }
+  }, [setLoading, setError, service]);
+
```

Comment:
Do you think it would be useful to put the count in `data` as well? Or do you think it's not necessary?

Reply:
IMO, it is not necessary for now because:
- it is not widely used across views . 
- it changes not often only when the `MarkFusionNotificationHistoryAsRead` is called (at least not now).
It might be good to reconsider to add it in `data` if we want to implement it in `react-card`